documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -0,0 +1,210 @@
+# Install the Clear Containers runtime on RHEL 7.3
+
+
+## Introduction
+
+Installing Clear Containers on RHEL is slightly more difficult than on other distributions mainly due to the fact that it does not provide current versions of gcc and glibc.
```

Comment:
For consistency with the rest of the doc, could you add backticks around `gcc` and `glibc`?